modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/internal/freemarker/tool/java/parser/ResourceOpenAPIParser.java

```diff
@@ -808,16 +810,12 @@ private static List<JavaMethodParameter> _getJavaMethodParameters(
 					"application/x-www-form-urlencoded is not supported");
 			}
 			else if (!requestBodyMediaTypes.contains("multipart/form-data")) {
```

Comment:
We could even make this condition reversed, if (multipart) the other code, else this code. Just a stupid comment, but it seems weird to me verifying a negative condition and if not, doing the opposite

Reply:
(This is just to improve readibility, but up to you)

Reply:
I can change it, I think it won't affect the code but I will rerun the bR just in case.